css/main.css

```diff
@@ -444,16 +504,38 @@ textarea {
     display: none;
   }
 
-  #desktop-nav {
-    display: block;
+  nav {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
   }
 
-  #desktop-nav ul {
+  .nav-menu {
     display: flex;
+  }
+
+  .nav-menu li {
+    border-bottom: none;
+  }
+
+  .menu-button {
+    display: none;
+  }
+
+  nav ul {
+    display: flex;
+    margin-left: auto;
     margin-right: 10vw;
   }
 
+  a .mail-icon {
+    display: block;
+  }
+
   .nav-link {
+    display: flex;
+    align-items: center;
     font-family: "Inter", sans-serif;
     font-style: normal;
     font-weight: 600;
```

Comment:
Please adjust the weight of your options font in the mobile menu to fit with the Figma design.

Figma options:
![design options](https://user-images.githubusercontent.com/42799579/190257911-ae81b37e-b42a-4403-bfe0-ecc79b97516d.png)

You options:
![user options](https://user-images.githubusercontent.com/42799579/190257914-a6b2fe78-066e-49c1-acf4-7e8e8edf1b1a.png)